src/app/pages/openauto.cpp

```diff
@@ -25,8 +26,12 @@ OpenAutoWorker::OpenAutoWorker(std::function<void(bool)> callback, bool night_mo
     this->create_io_service_workers();
 
     this->app->waitForDevice(true);
+    AAInterface* aa_interface = AAInterface::get_instance();
+    service_factory.setAndroidAutoInterface(aa_interface);
```

Comment:
[not for this pr] but this is a good example of where we could contain all setup-type stuff related to AA in a single spot (Session::AndroidAuto or wherever)